scanner/Makefile

```diff
@@ -325,12 +325,12 @@ $(certs-d)/%/cert.csr: $(certs-d)/%/key.pem
 .PRECIOUS: $(certs-d)/%/ca.pem
 $(certs-d)/%/ca.pem: $(certs-d)/ca/root.pem
 	@echo "+ $@"
-	$(SILENT)ln -fs $(abspath $<) $@
+	$(SILENT)cp $(abspath $<) $@
 
 .PRECIOUS: $(certs-d)/%/ca-key.pem
 $(certs-d)/%/ca-key.pem: $(certs-d)/ca/key.pem
 	@echo "+ $@"
-	$(SILENT)ln -fs $(abspath $<) $@
+	$(SILENT)cp $(abspath $<) $@
```

Comment:
Asking for my own understanding: why are we using `cp` instead now?

Reply:
More flexibility in case we want to run scanner or tests from within a build container. The symlink might not resolve if this is mounted as a volume.